src/relay.cpp

```diff
@@ -381,8 +383,13 @@ void prepare_socket(int *local_sock, int *server_sock, relay_config *config, int
     struct ifaddrs *ifa, *ifa_tmp;
     sockaddr_in6 addr;
     sockaddr_in6 ll_addr;
+#ifdef __STDC_LIB_EXT1__
+    memset_s(&addr, sizeof(addr), 0, sizeof(addr));
```

Comment:
>memset_s(&addr, sizeof(addr), 0, sizeof(addr)); [](http://example.com/codeflow?start=4&length=47)

For this special case (initialization of a local var), could you check this solution?
```
sockaddr_in6 addr = {0};
```

Reply:
Yes, please review